src/env/tock/storage.rs

```diff
@@ -100,6 +101,10 @@ fn block_command(driver: usize, cmd: usize, arg1: usize, arg2: usize) -> Storage
     }
 }
 
+fn read_slice(address: usize, length: usize) -> &'static [u8] {
```

Comment:
This function is unsafe without preconditions. We should either mark it `unsafe` to make call-site aware that there is a precondition to check, or document the precondition and hope it's not ignored.

```rust
/// Provides read-access to a slice from flash.
///
/// # Safety
///
/// The slice must not be modified during the lifetime of the program. For example, it should not point to the storage.

Reply:
I made the function unsafe.